spec/support/webmock.rb

```diff
@@ -1,3 +1,3 @@
 require 'webmock/rspec'
 app_host = ENV.fetch('CAPYBARA_SERVER_HOST', ENV.fetch('HOSTNAME', `hostname`.strip))
-WebMock.disable_net_connect!(allow: [app_host], allow_localhost: true)
+WebMock.disable_net_connect!(allow: [app_host, 'api.et.127.0.0.1.nip.io'], allow_localhost: true)
```

Comment:
This test suite should not rely on the full system being present, so no need to allow access to it from the tests.  In fact, better to not allow it so you can see anything that accidentally makes a request to it

Reply:
Removed in 24c3403